CHANGES.md

```diff
@@ -2,6 +2,7 @@
 # 0.11.0 (Unreleased)
 
 - [FIX] Using MongoDB query as the "gold" standard, query support for `NOT` has been fixed to return result sets correctly (as in MongoDB query).  Previously, the result set from a query like `{ "pet": { "$not" { "$eq": "dog" } } }` would include a document like `{ "pet" : [ "cat", "dog" ], ... }` because the array contains an object that isn't `dog`.  The new behavior is that `$not` now inverts the result set, so this document will no longer be included because it has an array element that matches `dog`.
+- [BREAKING CHANGE] Changed local document APIs, removed createLocalDocument and updateLocalDocument, replaced with insertLocalDocument. getLocalDocument return type changed to LocalDocument see commit for full details.
```

Comment:
Break this up into a few more sentences, eg

Changed local document APIs. createLocalDocument and updateLocalDocument have been removed and replaced by insertLocalDocument. The return type of getLocalDocument has been changed to LocalDocument.

Also see commit for full details is a bit vague - maybe refer to the specific class etc which has changed?